rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
```

Comment:
- [ ] Please make `book` and `person` as well `attr_accessor` to be able to access and modify them